### DISCUSSION TITLE
[Script request] Passbolt

### DISCUSSION THREAD

**Discussion by Depechie** [community-scripts/ProxmoxVE#1079] [Request script] [answered]
### Application Name

Passbolt

### Website

https://www.passbolt.com/ce/docker

### Description

Passbolt is an open source credential platform for modern teams. A versatile, battle-tested solution to manage and collaborate on passwords, accesses, and secrets.
The Community Edition is free.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Kiritzai** [comment]
I second this! Was searching for a good alternative for Vaultwarden en Passbolt is actually a solid alternative!

**Comment by tremor021** [comment]
<img width="1902" height="1015" alt="image" src="https://github.com/user-attachments/assets/6af1ecb0-2b29-4f10-905a-8710c01a924e" />

I already have a script for this... Just havent pushed to testing yet

**Comment by tremor021** [comment, marked as answer]
Done with #9226
